docs/sphinx/user_guide/feature/policies.rst

```diff
@@ -637,34 +637,36 @@ type. Atomic policy types are distinct from loop execution policy types.
-                          policy        kernel.
-cuda/hip_atomic_explicit  any CUDA/HIP  Atomic operation performed in a CUDA/HIP
-                          policy        kernel that may also be used in a host
-                                        execution context. The atomic policy
-                                        takes a host atomic policy template
-                                        argument. See additional explanation 
-                                        and example below.
-builtin_atomic            seq_exec,     Compiler *builtin* atomic operation.
-                          loop_exec,
-                          any OpenMP
-                          policy
-auto_atomic               seq_exec,     Atomic operation *compatible* with loop
-                          loop_exec,    execution policy. See example below.
-                          any OpenMP    Can not be used inside cuda/hip
-                          policy,       explicit atomic policies.
-                          any CUDA/HIP
-                          policy
-========================= ============= ========================================
+============================= ============= ========================================
+Atomic Policy                 Loop Policies Brief description
+                              to Use With
+============================= ============= ========================================
+seq_atomic                    seq_exec,     Atomic operation performed in a
+                              loop_exec     non-parallel (sequential) kernel.
+omp_atomic                    any OpenMP    Atomic operation performed in an OpenMP.
+                              policy        multithreading or target kernel; i.e.,
+                                              apply ``omp atomic`` pragma.
+cuda/hip/sycl_atomic          any           Atomic operation performed in a
+                              CUDA/HIP/SYCL CUDA/HIP/SYCL kernel.
+                              policy        
+cuda/hip/sycl_atomic_explicit any           Atomic operation performed in a
+                              CUDA/HIP/SYCL CUDA/HIP/SYCL kernel that may also be
+                              policy        used in a host execution context. The
+                                            atomic policy takes a host atomic 
+                                            policy template argument. See additional
+                                            explanation and example below.
+builtin_atomic                seq_exec,     Compiler *builtin* atomic operation.
+                              loop_exec,
+                              any OpenMP
+                              policy
+auto_atomic                   seq_exec,     Atomic operation *compatible* with loop
+                              loop_exec,    execution policy. See example below.
+                              any OpenMP    Can not be used inside cuda/hip
```

Comment:
Is this last sentence true for SYCL also?

Reply:
I tested it and no it is not.  I will update the documentation.  When these tests are enabled, they compile with SYCL but fail at run time.